assets/js/googlesitekit/api/cache.private.js

```diff
@@ -0,0 +1,113 @@
+let storageBackend;
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @param {*} backend Backend to set for the cache.
+ */
+export const setSelectedStorageBackend = ( backend ) => {
+	storageBackend = backend;
+};
+
+export const StorageKeyPrefix = 'googlesitekit_';
```

Comment:
I think we can capitalize this variable name to indicate it's a constant in code consuming it (`STORAGE_KEY_PREFIX`). Gutenberg does that too.